File: java-maven-jpa-static/src/main/java/pl/exsio/querydsl/entityql/examples/jpa/entity/GroupAdmin.java
package pl.exsio.querydsl.entityql.examples.jpa.entity;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name = "GROUP_ADMINS")
public class GroupAdmin {

    @Id
    @Column(name = "GA_ID")
    private Long id;

    @Column(name = "NAME")
    private String name;


    public GroupAdmin() {
    }

    public GroupAdmin(String name) {
        this.name = name;
    }
}

File: kotlin-maven-jpa-static/src/main/kotlin/pl/exsio/querydsl/entityql/examples/jpa/entity/GroupAdmin.kt
package pl.exsio.querydsl.entityql.examples.jpa.entity

import javax.persistence.*

@Entity
@Table(name = "GROUP_ADMINS")
class GroupAdmin() {

    @Id
    @Column(name = "GA_ID")
    @GeneratedValue
    var id: Long? = null

    @Column(name = "NAME")
    var name: String? = null

    constructor(id: Long, name: String) : this() {
        this.id = id
        this.name = name
    }
}
